app/views/pages/privacy_policy.html.erb

```diff
@@ -22,7 +22,7 @@
       <li><%= t('.list_item2') %></li>
     </ul>
 
-    <br>
+    <br/>
```

Comment:
Good point I didn't know that - I ran htmlhint and it complained so I fixed it. Must be the way we have it configured.